### PR TITLE
Adding Chip style

### PR DESCRIPTION
**Pull request by orm109**
https://github.com/TeamWertarbyte/material-ui-chip-input/issues/184

### CODE REVIEW

File: src/ChipInput.js [hunk 1]
```diff
@@ -554,7 +555,7 @@ class ChipInput extends React.Component {
                 isFocused: dataSourceConfig ? (this.state.focusedChip && this.state.focusedChip[dataSourceConfig.value] === value) : (this.state.focusedChip === value),
                 handleClick: () => this.setState({ focusedChip: value }),
                 handleRequestDelete: () => this.handleDeleteChip(value, i),
-                defaultStyle: styles.defaultChip
+                defaultStyle: Object.assign(styles.defaultChip, chipStyle)
```
Comment (leMaik) [inline src/ChipInput.js]: Just a note here: This would modify `styles.defaultChip` every time `render` gets called.

**Comment by leMaik** [comment]
Thanks a lot for the PR, but unfortunately, this feature is already available by using the `componentRenderer` prop.

File: src/ChipInput.js (hunk 1 above)
Comment (orm109) [reply src/ChipInput.js]: good one
had to change the order `Object.assign(chipStyle, styles.defaultChip)`
or use empty object as first elem `Object.assign({}, chipStyle, styles.defaultChip)`

**Comment by orm109** [comment]
I guess you mean `chipRenderer` @leMaik 
but it seems too much for just changing the style

what would be the `chipRenderer` for changing style?
can you write your example , and maybe even add it to storybook

**Comment by leMaik** [comment]
@orm109 I agree that there might be a slight overhead for just changing the styles, but this approach is far more flexible. The example in the storybook even changes the style. https://github.com/TeamWertarbyte/material-ui-chip-input/blob/master/stories/index.js#L104

**Pull request closed without merging**